source/jobs.erb

```diff
@@ -98,34 +98,38 @@ related:
     <h2 class="jobs-page-roles__title">
       <%= current_page.data.roles.title %>
     </h2>
-
-    <% current_page.data.roles.content.each do |role| %>
-      <% if role.link %>
-        <!--we need to have this extra class to Safari bug - it's overriding anchor color and makes a border purple in the nested element-->
-        <a class="jobs-page-tile--linked" href="<%= role.link %>" title="<%= role.name %>">
-          <div class="jobs-page-tile__content">
-            <img class="jobs-page-tile__image" src="<%= role.image %>" alt="<%= role.name %>">
-            <h3 class="jobs-page-tile__title">
-              <%= role.name %>
-            </h3>
-            <p class="jobs-page-tile__paragraph">
-              <%= role.text %>
-            </p>
-          </div>
-        </a>
-       <% else %>
-        <div class="jobs-page-tile">
-          <div class="jobs-page-tile__content--inactive">
-            <img class="jobs-page-tile__image" src="<%= role.image %>" alt="<%= role.name %>">
-            <h3 class="jobs-page-tile__title">
-              <%= role.name %>
-            </h3>
-            <p class="jobs-page-tile__paragraph">
-              <%= role.text %>
-            </p>
-          </div>
-        </div>
-      <% end %>
+    <% if current_page.data.roles.content.length == 1 %>
+      <% role = current_page.data.roles.content[0] %>
+          <a class="jobs-page-tile__content-single" href="<%= role.link %>" title="<%= role.name %>">
+              <div class="component__tile jobs-page-tile__text-single" >
+                <h3 class="jobs-page-tile__title">
+                  <%= role.name %>
+                </h3>
+                <p class="jobs-page-tile__paragraph">
+                  <%= role.text %>
+                </p>
+              </div>
+              <div class="jobs-page-tile__image-container">
+                <%= image_tag role.image, alt: role.name, class: "jobs-page-tile__image", srcset: retina_srcset(role.image) %>
+              </div>
+          </a>
+    <% else %>
+      <div class="jobs-page-tile-container">
```

Comment:
We could consider calling this `.jobs-page-tiles` in a similar way to how we have `.homepage-services` containing `.homepage-service` on the homepage